[llvm] build failure

It seems that not installing the package-default clang leads to problems in the (?install?) step.

**Host Environment**
- OS: Windows 10, 1903
- Compiler: MSVC 19.25.28614.0


**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg.exe install --triplet x64-windows-static llvm[core,disable-assertions,target-arm,target-aarch64,target-mips,target-powerpc,target-sparc,target-x86]`

**Failure logs** 
```
Starting package 7/9: llvm:x64-windows-static
Building package llvm[core,disable-assertions,target-aarch64,target-arm,target-mips,target-powerpc,target-sparc,target-x86]:x64-windows-static...
-- Downloading https://github.com/llvm/llvm-project/archive/llvmorg-10.0.0.tar.gz...
-- Extracting source E:/Projects/rehex/vcpkg/vcpkg/downloads/llvm-llvm-project-llvmorg-10.0.0.tar.gz
-- Applying patch 0001-allow-to-use-commas.patch
-- Applying patch 0002-fix-install-paths.patch
-- Applying patch 0003-fix-vs2019-v16.6.patch
-- Using source at E:/Projects/rehex/vcpkg/vcpkg/buildtrees/llvm/src/org-10.0.0-32c002c70c
-- Downloading https://www.python.org/ftp/python/3.8.3/python-3.8.3-embed-amd64.zip...
-- Configuring x64-windows-static
-- Building x64-windows-static-dbg
-- Building x64-windows-static-rel
CMake Error at scripts/cmake/vcpkg_fixup_cmake_targets.cmake:116 (message):

  'E:/Projects/rehex/vcpkg/vcpkg/packages/llvm_x64-windows-static/debug/share/clang'
  does not exist.
Call Stack (most recent call first):
  ports/llvm/portfile.cmake:201 (vcpkg_fixup_cmake_targets)
  scripts/ports.cmake:76 (include)


Error: Building package llvm:x64-windows-static failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: llvm:x64-windows-static
  Vcpkg version: 2020.02.04-nohash

Additionally, attach any relevant sections from the log files above.
```

Some relevant files from `E:\Projects\rehex\vcpkg\vcpkg\buildtrees\llvm`:
[stdout-x64-windows-static.log](https://github.com/microsoft/vcpkg/files/4774575/stdout-x64-windows-static.log)
[x64-windows-static.vcpkg_abi_info.txt](https://github.com/microsoft/vcpkg/files/4774577/x64-windows-static.vcpkg_abi_info.txt)

--- a/ports/llvm/CONTROL
+++ b/ports/llvm/CONTROL
@@ -1,5 +1,5 @@
 Source: llvm
-Version: 10.0.0-3
+Version: 10.0.0-4
 Homepage: https://llvm.org/
 Description: The LLVM Compiler Infrastructure
 Supports: !uwp

--- a/ports/llvm/portfile.cmake
+++ b/ports/llvm/portfile.cmake
@@ -1,5 +1,3 @@
-include(vcpkg_common_functions)
-
 set(VERSION "10.0.0")
 
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
@@ -197,7 +195,7 @@ vcpkg_configure_cmake(
 )
 
 vcpkg_install_cmake()
-vcpkg_fixup_cmake_targets(CONFIG_PATH share/llvm)
+vcpkg_fixup_cmake_targets(CONFIG_PATH share/${PORT})
 if("clang" IN_LIST FEATURES)
     vcpkg_fixup_cmake_targets(CONFIG_PATH share/clang TARGET_PATH share/clang)
 endif()
@@ -254,7 +252,7 @@ if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
 endif()
 
 # Handle copyright
-file(INSTALL ${SOURCE_PATH}/llvm/LICENSE.TXT DESTINATION ${CURRENT_PACKAGES_DIR}/share/llvm RENAME copyright)
+file(INSTALL ${SOURCE_PATH}/llvm/LICENSE.TXT DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
 if("clang" IN_LIST FEATURES)
     file(INSTALL ${SOURCE_PATH}/clang/LICENSE.TXT DESTINATION ${CURRENT_PACKAGES_DIR}/share/clang RENAME copyright)
 endif()

--- a/ports/llvm/portfile.cmake
+++ b/ports/llvm/portfile.cmake
@@ -198,7 +198,9 @@ vcpkg_configure_cmake(
 
 vcpkg_install_cmake()
 vcpkg_fixup_cmake_targets(CONFIG_PATH share/llvm)
-vcpkg_fixup_cmake_targets(CONFIG_PATH share/clang TARGET_PATH share/clang)
+if("clang" IN_LIST FEATURES)
+    vcpkg_fixup_cmake_targets(CONFIG_PATH share/clang TARGET_PATH share/clang)
+endif()
 
 if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
     file(GLOB_RECURSE _llvm_release_targets

